src/features/user/ManageProjects/components/BasicDetails.tsx

```diff
@@ -700,8 +700,8 @@ export default function BasicDetails({
               style={{
                 backgroundColor:
                   theme === 'theme-light'
-                    ? themeProperties.light.light
-                    : themeProperties.dark.dark,
+                    ? themeProperties.designSystem.colors.white
+                    : themeProperties.designSystem.colors.coreText,
```

Comment:
I suggest leaving the dark mode color as is, so that we can search for places where dark mode has been implemented and make adjustments later.